packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json

```diff
@@ -1,6 +1,5 @@
 {
   "pci_projects_project_storages_cold_archive_breadcrumb": "Cold Archive",
-  "pci_projects_project_storages_cold_archive_title": "Cold Archive",
```

Comment:
any raison to delete it ? it's required here `/cold-archive/cold-archive.html`:
`
data-heading="{{:: 'pci_projects_project_storages_cold_archive_title' | translate }}"
`

Reply:
Done.